### PR TITLE
LL-5234 (OperationDetails): sidedrawer design update + rework

### PR DESCRIPTION
**Pull request by LFBarreto**
- switch operation details to a drawer
- updated accounts testst
- update screenshots
- display image changed all the time
- correctly split things
- feedback
- update snapshots
- update drawer size
- update view in explorer button
- move padding to drawer content
- update margins
- touch header
- header style
- (OperationDetails): sidedrawer design update + rework

https://user-images.githubusercontent.com/11752937/119141663-d42a8c00-ba45-11eb-9007-d1d6d07703f8.mp4

localhost_8080_webpack_index html_theme=lightu
localhost_8080_webpack_index html_theme=lighty
localhost_8080_webpack_index html_theme=light____
localhost_8080_webpack_index html_theme=null

### Type

Feature
### Context

[LL-5234]
### Parts of the app affected / Test plan

Operation details side drawer + side drawers in general have been reworked
Look into all operation type cases:
- classic send / receive
- token operations
- navigation between amount info and operations
- extra data operations (wallet connect, lending ...)
- staking operations (tron, algorand, cosmos, polkadot)
- operations with multiple adresses
- swap operation details
- try and stress test the navigation animations a bit

[LL-5234]: https://ledgerhq.atlassian.net/browse/LL-5234

### CODE REVIEW

**Review by machard — APPROVED** [review]

**Review by machard — DISMISSED** [review]

**Review by Arnaud97234 — CHANGES_REQUESTED** [review]
See Linked task:
https://ledgerhq.atlassian.net/browse/LL-5234

File: src/renderer/drawers/Provider.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+type State = {
+  Component: ?React$ComponentType<*>,
+  props?: *,
+  open: boolean,
+};
+
+// actions
+// it makes them available and current from connector events handlers
+export let setDrawer: (Component?: *, props?: *) => void = () => {};
```
Comment (rjarasson-ledger) [inline src/renderer/drawers/Provider.js:12]: Why not providing a hook here ?

like:
```
export const useDrawer = () => {
  const context = useContext(DrawerContext);
  if (context === undefined) {
    throw new Error(`useDrawer must be used within a DrawerProvider`);
  }
  return context;
};
```

The solution here is a bit hackish i think.
And we can also avoid exporting the context directly (only provide it through the hook).

**Comment by juan-cortes** [comment]
Why does the drawer slide from right to left when you go _back_? 
cc video

File: src/renderer/drawers/Provider.js (hunk 1 above)
Comment (LFBarreto) [reply src/renderer/drawers/Provider.js:12]: As discussed not going to change this for now. :bow:

**Review by Arnaud97234 — CHANGES_REQUESTED** [review]
@LFBarreto This one is still occurring https://ledgerhq.atlassian.net/browse/LL-5815

**Review by Arnaud97234 — APPROVED** [review]
QA ✅ 
Waiting for the UI test to run then we can review and merge.

File: .github/actions/prepare-comment-body/index.js [hunk 2]
```diff
@@ -63,7 +63,7 @@ ${str}
 </details>
 `;
 
-  if (!lintFailed && !testsFailed && !imgDiffFailed && imgChanged.length) {
```
Comment (gre) [inline .github/actions/prepare-comment-body/index.js:66]: why are we changing this in this PR?

**Review by gre — APPROVED** [review]

**Pull request merged**